relay-dynamic-config/src/defaults.rs

```diff
@@ -766,6 +766,57 @@ fn span_metrics(transaction_extraction_enabled: bool) -> impl IntoIterator<Item
                     .always(),
             ],
         },
+        MetricSpec {
+            category: DataCategory::Span,
+            mri: "g:spans/messaging.message.body.size@byte".into(),
+            field: Some("span.measurements.messaging.message.body.size".into()),
+            condition: Some(is_queue_op.clone()),
+            tags: vec![
+                Tag::with_key("environment")
+                    .from_field("span.sentry_tags.environment")
+                    .always(),
+                Tag::with_key("span.op")
+                    .from_field("span.sentry_tags.op")
+                    .always(),
+                Tag::with_key("transaction")
+                    .from_field("span.sentry_tags.transaction")
+                    .always(),
```

Comment:
Just asking for good measure: Are we sure we need the transaction tag on all three of these metrics?

Reply:
`messaging.message.receive.latency` requires `transaction` since we will break down by transaction in the product workflow.

I revisited the mocks, and realized that `messaging.message.body.size` and `messaging.message.retry.count` are actually not used in any aggregate views, so we probably don't need them as metrics. I've removed them from the metrics extraction, but I am keeping them in the tag extraction since we still need them for indexed spans.